ui/src/server_axum.rs

```diff
@@ -160,6 +160,19 @@ pub(crate) async fn serve(config: Config) {
         .await
         .unwrap();
 
+    let socket_addr = &listener
+        .local_addr()
+        .expect("Unable to fetch local `SocketAddr`");
+    tracing::info!(
+        "Serving playground backend at {}:{} [{}]",
+        &socket_addr.ip(),
+        &socket_addr.port(),
+        (match &socket_addr.is_ipv4() {
+            true => "IPv4",
+            _ => "IPv6",
+        })
```

Comment:
Can you explain a little bit more about why you want to know this information? IPv4 and IPv6 addresses are very visually different. IPv4 looks like `127.0.0.1:5000` and IPv6 looks like `[::]:6000`. In what case would a person need to know if it's IPv4 vs IPv6 but not already know the difference in formatting?

Reply:
Hmm good point.  I've seen some (other) services sometimes starting up on an ipv6 interface and this has the potential of confusing someone who isn't aware, but as you pointed out, that would be unlikely.

I'll make the update, thanks.